src/ui/theme.rs

```diff
@@ -152,9 +152,14 @@ pub fn load_system_fonts(mut fonts: FontDefinitions) -> FontDefinitions{
         "MS Gothic",
         "Noto Sans SC",
         "PingFang SC",
+        "Source Han Sans CN",
```

Comment:
This logic will load the first match. If you list different fonts here, they are mutually exclusive.